utils/services/network/RtdbModules.ts

```diff
@@ -74,15 +73,31 @@ export async function rtdbArticleAddMirror(
     thumbnail,
     title,
     topics,
-    author: { 
-      id:user.id,
-      name:user.name,
-      username:user.username,
+    author: {
+      id: user.id,
+      name: user.name,
+      username: user.username,
       avatar: user.avatar,
-     },
+    },
   };
   // console.log(data);
   const newRef = ref(db, path);
   await set(newRef, data);
   return article;
 }
+
+export async function rtdbTopicGet(): Promise<string[]> {
+  const path = `topicList`;
+  // const query = child(ref(db),path);
+  const rr = ref(db, path);
+  const qq = query(rr, orderByValue());
```

Comment:
Mmmm i stuck at the orderByValue at the rtdb. but i think we can proceed since it is not a big deal anyway.

Reply:
Okay, will create a new issue! for it #21